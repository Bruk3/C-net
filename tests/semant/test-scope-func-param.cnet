int main()
{
    int z = mul(2, x); 
    return 0;
}
int add(int a, int b) {return a + b;}
int x = 1;
int mul(int x, int y){return x * y;}
