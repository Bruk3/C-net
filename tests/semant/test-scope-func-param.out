int main()
{
int z = (int : mul((int : 2), (int : x)));
return (int : 0);
}

int add(int a,int b)
{
return (int : (int : a) + (int : b));
}

int x = (int : 1);
int mul(int x,int y)
{
return (int : (int : x) * (int : y));
}

